2996_reflection/reflection.md

```diff
@@ -4315,15 +4325,27 @@ Add a new paragraph to the end of the section defining _data member description_
 
 ::: std
 ::: addu
-[29+]{.pnum} A _data member description_ is a quintuple (`$T$`, `$N$`, `$A$`, `$W$`, `$NU$`) describing the potential declaration of a nonstatic data member where
+[29+]{.pnum} A _data member description_ is a quintuple (`$T$`, `$N$`, `$A$`, `$W$`, `$NUA$`) describing the potential declaration of a nonstatic data member where
 
 - [29+.#]{.pnum} `$T$` is a type or type alias,
 - [29+.#]{.pnum} `$N$` is an `$identifier$` or `-1`,
 - [29+.#]{.pnum} `$A$` is an alignment or `-1`,
 - [29+.#]{.pnum} `$W$` is a bit-field width or `-1`, and
-- [29+.#]{.pnum} `$NU$` is a boolean value that is `true` if the potentially-declared member would be declared with the attribute `no_unique_address` and `false` otherwise.
+- [29+.#]{.pnum} `$NUA$` is a boolean value.
 
-Two data member descriptions are equal if each of their respective components are same types, same identifiers, or equal values.
+Two data member descriptions are equal if each of their respective components are same types, same identifiers, and equal values.
+
+::: note
+The components of a data member description describe a data member such that
+
+- [29+.#]{.pnum} its type is specified using the type or type alias given by `$T$`,
```

Comment:
We don't really do sub-paragraph numbering in notes.

Reply:
https://eel.is/c++draft/temp.res.general#note-4

I did check.

Reply:
Huh. Ok.